### ISSUE TITLE
Wrong Geo coordinates for DE-Hag4

### ISSUE THREAD

**Issue by peterreimer** [hbz/lobid-organisations#74]
One more:
Hagen -> Soltau http://lobid.org/organisation/DE-Hag4

**Comment by acka47** [comment]
Thanks for the pointer. I will go ahead and adjust the source data.

This time beta is correct:  http://beta.lobid.org/organisations/DE-Hag4, corrodinates in OSM: http://www.openstreetmap.org/?mlat=51.36562&mlon=7.49818#map=17/51.36562/7.49684 Looks like a reason to make it productive ASAP. Actually, we already consider it to be stable for the use cases in NWBib and use it there. Most probably, there should be no problem to use it in your case as well. Let's talk about it. I will try to catch you next week.

**Comment by dr0i** [comment]
>  Looks like a reason to make it productive ASAP

Don't want to speak against the switch in principle. But that shouldn't be too much motivated by the geo coordinates. I spoke with @SBRitter ,  and when we both remembered that right, the geo coordinates in beta-lobid-organisations are based on an old [serialized HashMap](https://github.com/lobid/lodmill/blob/master/lodmill-rd/src/main/java/org/lobid/lodmill/PipeLobidOrganisationEnrichment.java#L102) created (and updated) by the old lobid-organisation code. It's a static file, thus, when a new beta.lobid-organisation resource  is added lacking geo coordinates in the data and in the static geo coord file  (see http://lobid.org/download/lookup-tables/data/ -> geo-l* ) the resource will have no geo coordinate at all, I guess (unlike lobid-organisation which will make a lookup dynamically, and ,  Iin fact updated  the database (the HashMap) just 2 hours ago))).

But I have no explanation why the more-up-to date hashMap of lobid-organisation is wrong and the old version is , if not always right, in principle better. I remember that at one point the nominatim API was changed and the same lookup results in *lesser* quality and I had to change the lookup URL.  Maybe the bad data is an heir of these times.

However, future hope lies in the upcoming geo lookup microservice to be implemented by @philboeselager which comes with versioning (I think) and will be more curated, hopefully.
